community-operators/service-binding-operator/0.0.23/service-binding-operator.v0.0.23.clusterserviceversion.yaml

```diff
@@ -18,7 +18,58 @@ spec:
       kind: ServiceBindingRequest
       name: servicebindingrequests.apps.openshift.io
       version: v1alpha1
-  description: Placeholder description
+  description: '
+  The Service Binding Operator enables users to create a managed binding between applications and operator-backed services such as databases, without having to perform manual configuration of secrets,  configmaps etc.
```

Comment:
The same edits should also be made to this file. Thx!

Reply:
Done